psi4/src/psi4/libcubeprop/csg.cc

```diff
@@ -649,6 +649,33 @@ void CubicScalarGrid::compute_orbitals(std::shared_ptr<Matrix> C, const std::vec
     }
     free_block(v);
 }
+void CubicScalarGrid::compute_difference(std::shared_ptr<Matrix> C, const std::vector<int>& indices,
+                                         const std::string& label, bool square, const std::string& type) {
```

Comment:
Best to make this `const bool square`, the compiler will likely optimize the loop below.